### DISCUSSION TITLE
Shaw/Rogers "Upgrading" Cable Boxes in Canada

### DISCUSSION THREAD

**Discussion by Striker-WG** [OpenSageTV/Community#87] [Tuners]
So I've been contacted by Shaw/Rogers that they're phasing out my current cable boxes (DCX-3200) and I will need to upgrade to one of their new units.

Looking at this page: https://support.shaw.ca/t5/tv-articles/equipment-info-shaw-tv-boxes/ta-p/5181

It looks like none of their current offerings provide Component connections for analog HD, so my HD-PVR's are no good, and I'm guessing my 1st gen Colossus cards aren't any good either.

I'm thinking this might be the final nail in the coffin for my SageTV setup.  I've seen there are some network encoder options, but i'm also not sure about channel tuning with these new STB's as there is no Firewire ports and i'm unclear if they are IR based remotes or Bluetooth....

Has anyone dealt with this yet and had any successes?

**Comment by Sagetvmaster** [comment]
Shaw Direct satellite has done the same thing.  Sucks.  Had to get a little HDMI box that strips HDCP crap, and then that feeds my Colossus card.  Channel change is IR using either the Hauppauge blaster or USB-UIRT device.  Channel changes seem slower by about five seconds, but it works.  Rogers blows, but not enough for me to leave Sage, STILL the best, software based DVR.  By a long shot.  Because it records live tv.
This thing works too if your dead-set on feeding component to your tuners:

https://www.amazon.com/LiNKFOR-1080P-Component-Converter-Adapter/dp/B07PMGBVV8/ref=cm_cr_arp_d_product_top?ie=UTF8

This guy breaks HDCP too I believe:

https://www.amazon.com/gp/product/B004F9LVXC/ref=ppx_yo_dt_b_asin_title_o09_s00?ie=UTF8&psc=1

**Comment by wayner9** [comment]
I am pretty sure that there will be IR options.  Most boxes can still be controlled via IR, even if their main remote uses BT.

The other thing to be aware of is you may need to make some changes to your home network if you use your own router rather than the router from Shaw as these IPTV cable boxes use your LAN to connect back to the main unit, rather than RF.

There are a bunch of threads on the network encoders - originally they were called the BM1000 or something like that.  Now the most popular units are buy Uray.  Once you get it working it should be pretty good as it doesn't require local drivers on your PC, in that way it is like the HD HomeRun device.  I like those as they are very easy to move to another SageTV server if you need to do that.  But the HDMI to Component looks like a decent option as well.

I am dreading having to do the same thing.  I have Rogers cable in Toronto and I am pretty sure that it won't be long until they turn off legacy cable as well.  And I also use Firewire, although I have SA/Cisco boxes rather than Motorola.

**Comment by baudfather** [comment]
I just went through this here in BC as they dropped support on all 3 of my 32xx boxes.  Try and get on a chat or phone call with their tech support (sales team might be useless in this case).  Ask them to upgrade your boxes to the DCX3510-M - I believe it's one of the last Moto/Arri boxes they will be supporting (for a few more years hopefully).  Still has firewire.  Initially they said they would only upgrade me to their IPTV / ignite setups that was pointless having Sagetv.  After a bit of research and a helpful tech I found the 3510 was still supported and they appear to still have some inventory of those.  I was able to get 2 for free.  One of them captures video from firewire, the other is using openDCT via a BM1000 to capture the non-MPEG2 channels.  Can still get the BM1000 cheap on ebay or aliexpress and is more than sufficient for capture from these moto boxes.

**Comment by Striker-WG** [comment]
Thanks for the helpful comments everyone!  I'll reach out to Shaw and see if I can get some of these DCX3510-M's as that sounds like the simplest solution for me.

If they can't or won't do that for me, i'll look at some alternate recording options like the HDMI to Component adapters that were suggested.

If I do go that route and have to give up FireWire channel tuning, does anyone have any good solutions for IR tuning that would handle 4 STB's?  I don't think I can use the IR blaster built into my HDPVR's (if I remember correctly, you can at best utilize one in a mutli-HDPVR setup) and the USB-UIRT only handles 2 STB's (not sure if I can combine two of them to handle 4 STB's?).

**Reply by wayner9** [reply]
Are you on Windows?

If so then the USB-UIRTs can change channels in three zones.  There is a stereo mini-headphone jack that has two of the zone - tip is one zone and ring is another - the ground is common.  You will need IR buds for these.  And the front of the unit is the third zone.  But then you have to shield the IR inputs of the boxes connected to zone 1 and 2 so that they don't get blasted by zone 3 which is blasting the whole room.

I am pretty sure that you can use multiple USB-UIRTs on one PC.

On Linux/unRAID I am pretty sure that a USB-UIRT can only do one zone.

There is also a device called the Global Cache GC-100.  I bought a few of these used on eBay.  They have 6 IR outputs and there have been reports of them working with SageTV, if you can find that on the old forums.

The other thing that is possible is changing chanels via IP commands.  I am pretty sure that the Ignite (actually Comcast Xfinity) boxes can be controlled via IP.  If we knew how to do that then you should be able to use that, just how any third part command can change channels which is how things work with Firewire, at least in Windows.

**Comment by c5davey** [comment]
I did the upgrade a few months ago. Not sure if this is region dependent but I had to retire my DCX3200 phase 2 box but they replaced it with a DCX3200-M phase 3. I hope Shaw/Rogers doesn't retire these boxes anytime soon.

**Reply by wayner9** [reply]
That looks like a traditional cable box.  It is only a matter of time before legacy cable is shut down and we will all have to go to Ignite.  But that could be six months or it could be a few years.  They will likely do it in phases across the country.

**Comment by emveepee** [comment]
Just had a most frustrating time with Rogers all my cable boxes I use with HDMI or other capture boxes including the 4K PVR stopped working on many channels.  I checked with the old handly diagnostics Scientific Atlanta screen and the signal is 0db but anything on the 700MHhz spectrum now fails, it locks to one frequency.  Rogers will not even send anyone to diagnose this since only Ignite (the IP service).  I can get channels on the lower bands 100Mb and 500Mb only.  Time to cancel Rogers.
